src/views/PurchaseOrder.vue

```diff
@@ -145,6 +145,7 @@ export default defineComponent({
           showToast(translate("Map all fields"));
           return
         }
+        this.mappingName = "";
```

Comment:
When assigning empty value to the mappingName here, it will save the mapping with an empty name in localStorage.

Reply:
Yes sir. Fixed it.